examples/03_second_level_models/plot_second_level_two_sample_test.py

```diff
@@ -1,4 +1,4 @@
-"""Second-level fMRI model: a two-sample test 
```

Comment:
I think I would drop that one. It's not super informative WDYT ?

Reply:
Yeah, basically 2 examples are sufficient to me.